mirage/factories/user.js

```diff
@@ -8,10 +8,24 @@ export default Factory.extend({
   email: () => faker.internet.email(),
   password: () => faker.internet.password(),
 
+  fullName() {
+    if (!this.firstName && !this.lastName) {
```

Comment:
Maybe use ` !this.firstName || !this.lastName` otherwise you get `Jonas ` or ` Cosandey`

Reply:
done